README.md

```diff
@@ -19,6 +19,43 @@ Navigate to the [latest release](https://github.com/rvr06/cornifer/releases/late
 
 Then, within VSCode, open the `Extensions` view, select for `Install from VSIX` from `Views and more actions` button and browse to the `.vsix`.
 
+To automate the installation of the extension in a devcontainer, follow these steps:
+
+1. Place the `.vsix` file in the repository under the `.devcontainer` directory, for example, `.devcontainer/extensions/cornifer-<version>.vsix`.
+
+2. Make the extension available to the devcontainer by adding it as a volume, e.g. in case you use compose add the extention folder als volume in the `docker-compose.yml`:
+
+```yaml
+version: '3'
+
+services:
+  app:
+    build: 
+      context: .
+      dockerfile: Dockerfile
+
+    volumes:
+      # Update this to wherever you want VS Code to mount the folder of your project
+      - ../..:/workspaces:cached
+      # Unpublished extensions
+      - ./extensions:/extensions
+```
+
+3. Finally, install the extension in the devcontainer.json:
+
+```json
+"customizations": {
+    "vscode": {    
+        "extensions": [
+            "/extensions/cornifer-<version>.vsix"
+        ]
+    }
+}
+```

```

Comment:
I had to switch `docker in docker` feature on as well to enable `Structurizr` preview, resulting in:
```json
{
	"dockerComposeFile": "compose.yaml",
	"service": "devcontainer",
	"workspaceFolder": "/workspaces/${localWorkspaceFolderBasename}",
	"customizations": {
		"vscode": {
			"extensions": [
				"/extensions/cornifer-0.30.0.vsix"
			]
		}
	},
	"features": {
		"ghcr.io/devcontainers/features/docker-in-docker:2.11.0": {}
	}
}
```

Reply:
Good point. I forgot to mention that I am using the docker-from-docker feature, which does also work:

"ghcr.io/devcontainers/features/docker-from-docker:1": {
			"version": "latest",
			"enableNonRootDocker": "true",
			"moby": "true"
		},